app/javascript/components/shared_components/PaginationPrevButton.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button';
+import { ArrowLeftIcon } from '@heroicons/react/outline';
+import { ArrowRightIcon } from '@heroicons/react/solid';
+import PropTypes from 'prop-types';
+
+export default function PaginationButton({ direction }) {
+  if (direction === 'Previous') {
+    return (
+      <Button variant="brand-backward">
```

Comment:
I think it's fine as it currently is.
This button style is used thoroughly, not only for pagination.

Reply:
Oh, It's just the naming scheme,
I was wondering if the word **backward** is adequate as a variant name, it gave me an insight that it's the variant for a backward (previous) button and not a generic button variant,
It's not an issue, it may be because I don't fully understand the meaning behind **backward**,

Reply:
Yeah, maybe "brand-outline" would be more adequate.